tests/conftest.py

```diff
@@ -229,6 +229,84 @@ async def guard_func(*args: _P.args, **kwargs: _P.kwargs) -> _R:
 location.async_detect_location_info = check_real(location.async_detect_location_info)
 
 
+# Test fixtures with autouse enabled and scope set to session.
```

Comment:
How do we know where the end of this part of the file is? Maybe move it to the bottom instead?